Manon-Seb/tp-space-x
# [Front] Launches view

## Task
Create launches components + services

## Type of change

## Description
- Add the launches component
- Add the launches services

## Testing scenario
- Launch docker: `docker-compose up`
- Launch the container: `docker exec space-x /bin/sh`
- Go to the url `localhost:4200`
- Go to the url `localhost:4200/launches` to see the launches list
